CHANGES.md

```diff
@@ -1,3 +1,18 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured.
```

Comment:
most of the previous release notes do not end with a `.`, consider removing for consistency. in the bullets with multiple sentences, maybe just split them with a `;`

Reply:
Ah whoops did not mean to include a `.` there. Thanks!